VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/InitialShape.cpp

```diff
@@ -300,7 +300,16 @@ UStaticMesh* CreateDefaultStaticMesh()
 	StaticMesh = NewObject<UStaticMesh>();
 #endif
 
-	StaticMesh->BuildFromMeshDescriptions(MeshDescriptions);
+	UStaticMesh::FBuildMeshDescriptionsParams Params;
+	Params.bCommitMeshDescription = true;

```

Comment:
according to the doc this is true by default, but we can leave it in for explicitly